### ISSUE TITLE
Env property during Jenkins build

### ISSUE THREAD

**Issue by dinesh76in** [authorjapps/zerocode#23]
@authorjapps , the below feature is needed for the some project teams if they are using the same test suites for different environments e.g. DEV, SIT, UAT, PRE-PROD etc.
Based on the env parameter value, the specific property file should be picked up by `zerocode` runner.
Is this is already supported in the framework?

**Comment by authorjapps** [comment]
@dinesh76in , this is being developed here as a PR https://github.com/authorjapps/zerocode/pull/24
Once this gets merged, you should be able to use the env property as below-

```
// if "ENV_NAME=ci", then it picks "ci_app_config.properties". 
// If ENV_NAME not set, then defaults to "app_config.properties" mentioned via @TargetEnv

@EnvProperty("${ENV_NAME}_") 
@TargetEnv("app_config.properties")
@RunWith(TestOnlyZeroCodeUnitRunner.class)
public class ZeroCodeEnvPropertyReaderTest {

    @Test
    @JsonTestCase("01_verification_test_cases/22_env_property_dynamic_runtime.json")
    public void testRunAgainstConfigPropertySetViaJenkins() throws Exception {        
    }
}
```

**Comment by authorjapps** [comment]
This has been released now to maven central, use version `1.1.22` or higher.
Search here for latest : http://search.maven.org/#search%7Cga%7C1%7Czerocode

For help on how to use, always refer to README. 

Click below for the help n usage of this particular issue-
[Setting Jenkins env propperty and picking environment specific properties file](https://github.com/authorjapps/zerocode#21)
